Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/entities/PatientResource.java

```diff
@@ -0,0 +1,35 @@
+package com.lantanagroup.link.measureeval.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import org.hl7.fhir.r4.model.ResourceType;
+
+import java.util.Date;
+
+@Entity
+public class PatientResource {
+    @Id
+    @GeneratedValue
+    private String id;
+
+    private String tenantId;
+
+    private String patientId;
+
+    private ResourceType resourceType;
+
+    private String resourceId;
+
+    private Types type;
+
+    private String resourceJson;
+
+    private Date timestamp;
+
+    public enum Types {
+        Original,
+        Normalized,
```

Comment:
Not necessarily. I'm leaning on handling that as part of implementation of the functionality, rather than addressing here in this "framework" setup. Happy to pivot/change the `PatientResourceStages` approach during implementation. Agreed that persisting both the original and normalized versions is turning out to be a lot of effort. But, we definitely need enough details in audit logs to be able to get _back_ to the original resource.